src/javascript/app/pages/user/account/change_password.js

```diff
@@ -1,45 +1,259 @@
+            $(frm_set_trading_password_error_id).text(response.error.message).setVisibility(1);
+            setTimeout(() => {
+                $(frm_set_trading_password_error_id).setVisibility(0);
+            }, 5000);
+        } else {
+            $msg_success_trading.text(localize('Your trading password has been set. Please use this to log in to MetaTrader 5 and Deriv X.'));
+            $msg_success_trading_container.setVisibility(1);
+            $trading_password_container.setVisibility(1);
+            $set_trading_password_container.setVisibility(0);
+            $(trading_form_id).trigger('reset');
+            setTimeout(() => {
+                $msg_success_trading.setVisibility(0);
+                BinarySocket.send({ get_account_status: 1 }).then(() => onLoad());
+            }, 5000);
+        }
+    };
+
+    const tradingPwHandler = (response) => {
+        if ('error' in response) {
+            const frm_change_trading_password_error_id = '#frm_change_trading_password_error';
+            $(frm_change_trading_password_error_id).text(response.error.message).setVisibility(1);
+            setTimeout(() => {
+                $(frm_change_trading_password_error_id).setVisibility(0);
+            }, 5000);
+        } else {
+            $msg_success_trading.text(localize('Your trading password has been changed. Please use this to log in to MetaTrader 5 and Deriv X.'));
+            $msg_success_trading_container.setVisibility(1);
+            $(trading_form_id).trigger('reset');
+            Password.removeCheck('#new_trading_password', true);
+            setTimeout(() => {
+                $msg_success_trading.setVisibility(0);
+            }, 5000);
+        }
+    };
+
     const onLoad = () => {
+        $change_password_container      = $('#change_password_container');
+        $change_password_loading        = $('#change_password_loading');
+        binary_form_id                  = '#frm_change_password';
+        trading_form_id                 = '#frm_trading_password';
+        social_unlink_btn_id            = '#social_unlink_button';
+        $binary_password_container      = $('#binary_password_container');
+        $social_signup_container        = $('#social_signup_container');
+        $trading_password_container     = $('#trading_password_container');
+        $set_trading_password_container = $('#set_trading_password_container');
+        social_signup_identifier        = '';
+        $msg_success_container          = $('#msg_success_container');
+        $msg_success                    = $('#msg_success');
+        $msg_success_trading_container  = $('#msg_success_trading_container');
+        $msg_success_trading            = $('#msg_success_trading');
+        forgot_binary_pw_btn_id         = '#forgot_binary_pw_btn';
+        forgot_trading_pw_btn_id         = '#forgot_trading_pw_btn';
+
+        $change_password_loading.setVisibility(1);
+
         BinarySocket.wait('get_account_status').then(() => {
```

Comment:
here we are already waiting for get account status. so we can remove the throw new error things. the check for array of status existing is fine and can stay.